types/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// TypeScript Version: 3.5
+
+import {Node, Data} from 'unist'
+
+declare namespace unistUtilRemovePosition {
+  interface NodeWithUndefinedPosition extends Node {
+    position: undefined
+  }
+
+  interface NodeWithoutPosition {
+    [key: string]: unknown
+    type: string
+    data?: Data
+  }
+}
+
+/* Utility to remove positions from a tree */
```

Comment:
Doc comments can be helpful for TS aware editors to show what params do, as they are typed.

```suggestion
/**
 * Utility to remove positions from a tree
 *
 * @param node remove position fields from `node`.
 * @param force if `force` is given, uses `delete`, otherwise, sets positions to `undefined`.
 */
```